python/package.json

```diff
@@ -12,20 +24,20 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/ccxt/ccxt.git"
+    "url": "https://github.com/carlosmiei/ccxt/tree/esm-migration"
```

Comment:
can we change this lol

Reply:
@frosty00 ofc 😅  I changed it because I needed to publish it to be consumed by ccxt.pro
https://www.npmjs.com/package/ccxt-esm